### ISSUE TITLE
Video continues playing in the when creating a video stream from a file

### ISSUE THREAD

**Issue by SylvainCorlay** [QuantStack/xwebrtc#14]
This may be the expected behavior, but I am not sure we should play the sound of the original video widget.

**Comment by SylvainCorlay** [comment]
ping @martinRenou does this ring a bell?

**Comment by martinRenou** [comment]
Yes, this is an issue with Chrome IIRC, it does not happen with Firefox, or maybe the other way around.

The issue is that we can hear the sound of the stream playing even though the videostream is paused on the page. The sound you hear is coming from the offscreen video element we use for syncing the stream views.

I remember opening a ticket to Chromium/Firefox but they said it was not an issue.
